### DISCUSSION TITLE
help with halloween decorations and person detection.

### DISCUSSION THREAD

**Discussion by stoneobscurity** [blakeblackshear/frigate#14394] [General]
i have setup some motion masks and person detect masks, but still getting alerted.

![image](https://github.com/user-attachments/assets/0ba512d0-5e79-4941-b2ef-784a7685c567)

![image](https://github.com/user-attachments/assets/e6aa4e97-81e9-43be-99f8-8c3e42fd531b)

![image](https://github.com/user-attachments/assets/43c1abbb-3bbb-42e1-871e-a110bbba6259)

![Screenshot 2024-10-16 at 12 51 12 PM](https://github.com/user-attachments/assets/3e52b0de-5f6a-4309-bde4-6fef313885eb)

![Screenshot 2024-10-16 at 12 51 22 PM](https://github.com/user-attachments/assets/a2ae67d1-33f9-4448-8117-40a9dd2e6405)

**Comment by NickM-27** [comment]
please read the docs on how to setup object masks, these have not been setup correctly. Also in general there is no need to use a motion mask for this https://docs.frigate.video/configuration/masks#object-filter-masks

**Comment by stoneobscurity** [comment]
are you talking about the small boxes inside the larger boxes in the first mask image? the small ones are the normal ones, the big ones are the new TEMPORARY ones for the ween.
